packages/typespec-test/test/openai_generic/generated/typespec-ts/src/api/models/index.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
For this file name, this operation are indeed inside an interface called `Models`. I feel like it's kind of confusing given models means where we put model types. Wonder if codegen should do something about it ?

Reply:
Unfortunately I don't think codegen needs to do anything about this unless there is a conflict with reserved words or words we use internally and cause a problem